app/models/miq_queue.rb

```diff
@@ -31,6 +31,26 @@ class MiqQueue < ApplicationRecord
   PRIORITY_WHICH  = [:max, :high, :normal, :low, :min]
   PRIORITY_DIR    = [:higher, :lower]
 
+  def self.artemis_events_client
```

Comment:
I don't think we should define a new client method when the only difference is the client_ref, maybe a general `artemis_client` that takes additional connect_opts?  Then we can define the client_ref in the event_handler.

Reply:
yeah, I thought so too, but as I'm caching I wanted to wait until others start using the client too - but I can quickly change that, no problem

Reply:
Yeah maybe define a method on the `EventHandler` so you don't have to pass the client_ref around everytime we call the client?